pipeline/builder/types/types.go

```diff
@@ -332,6 +332,7 @@ type Container struct {
 
 	LivenessProbe  *Probe `json:"livenessProbe"`
 	ReadinessProbe *Probe `json:"readinessProbe"`
+	StartupProbe   *Probe `json:"startupProbe"`
```

Comment:
neat! I'm glad to see this being added. We're going to need to do a bit more work to support this however. Basically, you'll need to handle mapping the startup probe's definition from the yml file over to the json payload that gets sent to spinnaker.

Reply:
Great, thanks for the guidance. I'll be making a Jira ticket to work that part then

Reply:
@petermcneely Where does the mapping happen? The startup probe shares the same structure as the others, fyi